string task_name
---
string task_state
bool success